packages/components/atoms/f-button/src/components/RouterLinkButton.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+    <router-link
+        :to="to">
+        <slot />
+    </router-link>
+</template>
+
+<script>
+export default {
+    name: 'RouterLinkButton',
+    props: {
+        to: {
+            type: String,
```

Comment:
The router link `to` attribute [can be either a string or a location object](https://router.vuejs.org/api/#to).